### DISCUSSION TITLE
Auto Throttle on Vision Jet

### DISCUSSION THREAD

**Discussion by russellmcg** [nguyenquyhy/Flight-Tracker-StreamDeck#414] [Q&A]
I am trying to set up a button where I can toggle on and off the auto throttle on the vision jet. I have it where if I press the button using "AUTO_THROTTLE_ARM" as the "Toggle Event", it will arm. Where I am stuck is finding the event that will trigger the on and off on the button. On my stream deck, it stays on the off image. 
![Screenshot 2023-12-13 063603](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/130863059/69861412-7b71-4cde-9bdd-b2d9f3c4e630)

**Comment by noblackthunder** [comment]
MOBIFLIGHT_SF50_AP_AT_Button_Press

<img width="929" alt="image" src="https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/1668416/bdc51599-c99d-4ef3-8ca0-504358ef19ed">

[FlightFXVisionjet SF50Air and honda jet Mobiflight events.txt](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/14028381/FlightFXVisionjet.SF50Air.and.honda.jet.Mobiflight.events.txt) (paste into WASDM event.txt) 

Need to use mobiflight but this should work ( default profiles you can find here https://hubhop.mobiflight.com/presets/) 

added the event text that i also use for all buttons. Not sure about the On OFF event view though
